NameGame/NameGame/Model/UserRecord/UserRecord.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  UserRecord.swift
+//  NameGame
+//
+//  Created by Tammy Le on 6/12/17.
+//  Copyright © 2017 WillowTree Apps. All rights reserved.
+//
+
+import Foundation
+
+struct UserRecord {
+    
+    let id: String
+    let firstName: String
+    let lastName: String
+    let headshot: Headshot
+    
+    private enum JSONKey: String {
```

Comment:
Style change; move the enum definition above the properties. Also for future compatibility with Swift 4 lets change the name from `JSONKey` to `CodingKeys`.